src/users/users.service.ts

```diff
@@ -63,4 +65,40 @@ export class UsersService {
     this.logger.log(message);
     return this.usersRepository.delete(id);
   }
+
+  //partner
+  async addPartner(requester: string, requestee: string): Promise<any> {
+    const token = {
+      partner_token: null,
+    };
+    const requestUser = await this.findByUsername(requester);
+    const targetUser = await this.findByUsername(requestee);
+    const message = `usersService.addPartner() requestUser=${requestUser.username} targetUser=${targetUser.username}`;
+    this.logger.log(message);
+
+    this.usersRepository.handlePartnerRequest(requestUser);
```

Comment:
What do you think to update `requestedAt` for `targetUser`?

Reply:
right, I forgot that we're not allowing users to send a request if someone else sends them one